typescript/src/redemption.ts

```diff
@@ -441,6 +441,7 @@ export async function findWalletForRedemption(
       }
     | undefined = undefined
   let maxAmount = BigNumber.from(0)
+  let activeWalletsCounter = 0
```

Comment:
There is always only one "active" wallet in the system. This is the one that accepts new deposits. What we are talking about here are "live" wallets that are operable from the Bridge's standpoint. I suggest naming this variable `liveWalletsCounter` and adjusting the nomenclature everywhere accordingly.

Reply:
19e2dff